ncbi/src/test/java/org/reactome/release/EuropePMCTest.java

```diff
@@ -0,0 +1,79 @@
+package org.reactome.release;
+
+import org.junit.jupiter.api.*;
+
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EuropePMCTest {
+
+	@Test
+	public void emptySetOfEuropePMCLinksFromEmptyDatabase() {
+		DummyGraphDBServer dummyGraphDBServer = DummyGraphDBServer.getInstance();
+		dummyGraphDBServer.initializeNeo4j();
+
+		Set<EuropePMC.EuropePMCLink> europePMCLinks = EuropePMC.fetchEuropePMCLinks(dummyGraphDBServer.getSession());
+
+		assertThat(europePMCLinks, is(empty()));
+	}
+
+	@Test
+	public void singleEuropePMCLinkFromMockGraphDB() {
+		DummyGraphDBServer dummyGraphDBServer = DummyGraphDBServer.getInstance();
+		dummyGraphDBServer.initializeNeo4j();
+		dummyGraphDBServer.populateDummyGraphDB();
+
+		EuropePMC.EuropePMCLink expectedEuropePMCLink = new EuropePMC.EuropePMCLink(
+			"p53-Dependent G1 DNA Damage Response",
+			"R-HSA-69563",
+			"9153395"
+		);
+
+		Set<EuropePMC.EuropePMCLink> europePMCLinks = EuropePMC.fetchEuropePMCLinks(dummyGraphDBServer.getSession());
+
+		assertThat(europePMCLinks, contains(expectedEuropePMCLink));
```

Comment:
How is this different from using assertTrue, as in `assertTrue(europePMCLinks.contains(expectedEuropePMCLink))`? Actually, where is `contains` even defined?

Reply:
`assertThat` and `contains` are methods from the Hamcrest library.  

`assertThat` gives more detail on failure than `assertTrue` in terms of what was expected and what was found during the test.  It's very useful for diagnosing the failure and fixing the issue.

`contains` is a matcher that looks for exactly the provided values occurring in the collection given in the first parameter.